maqs-webservices/src/main/java/com/cognizantsoftvision/maqs/webservices/WebServiceTestObject.java

```diff
@@ -5,14 +5,15 @@
 package com.cognizantsoftvision.maqs.webservices;
 
 import com.cognizantsoftvision.maqs.base.BaseTestObject;
-import com.cognizantsoftvision.maqs.utilities.logging.Logger;
+import com.cognizantsoftvision.maqs.utilities.logging.ILogger;
 import java.net.http.HttpClient;
 import java.util.function.Supplier;
 
 /**
  * The Web Service Test Object class.
  */
-public class WebServiceTestObject extends BaseTestObject {
+public class WebServiceTestObject extends BaseTestObject implements IWebServiceTestObject {
```

Comment:
Why are we implementing the base service interface if it is not required for the tests?